docker/prod/nodejs/nginx_staging.conf

```diff
@@ -64,6 +72,20 @@ server {
     proxy_pass http://statsd_exporter;
   }
 
+  location /nodeexporter {
+    proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+    proxy_set_header Host $host;
+    proxy_set_header X-Forwarded-Proto $scheme;
+    proxy_pass http://node_exporter/metrics;
```

Comment:
be consistent with your naming convention. At some place you are using `node_exporter` and at other places `nodeexporter`. This is not a good practice